spec/cucumber/runtime_spec.rb

```diff
@@ -20,5 +20,26 @@ module Cucumber
         expect(subject.doc_string('Text')).to eq 'Text'
       end
     end
+
+    describe '#install_wire_plugin' do
+      it 'informs the user it is deprecated' do
+        stub_const('Cucumber::Deprecate::STRATEGY', Cucumber::Deprecate::ForUsers)
+        allow(STDERR).to receive(:puts)
+        allow_any_instance_of(Configuration).to receive(:all_files_to_load).and_return(['file.wire'])
+
+        begin
+          subject.run!
```

Comment:
There's probably a better way of doing this, but if we're deleting it soon it's not worth the hassle

Reply:
Yes, this won't last for long

I'm curious: what do you have in mind?